bluepyemodel/access_point/export_emodel.py

```diff
@@ -0,0 +1,147 @@
+"""Export the emodels to Nexus"""
```

Comment:
Shouldn't we put this file in bluepyemodel/export_emodel? I don't have a strong opinion about this, I'm just raising this point.

Reply:
Good point, yes I think It make sense to move it there